api/index.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express')
+const morgan = require('morgan')
+const cors = require('cors')
+const Parse = require('parse/node')
+const route = require('./authentication/authentication')
+//import Parse from 'parse/dist/parse.min.js';
+const {PARSE_APP_ID, PARSE_JAVASCRIPT_KEY} = require('./config')
+//const { default: App } = require('../ui/src/components/App/App')
+
+const app = express()
+const port = process.env.PORT || 3001
+
+app.use(express.json())
+app.use(morgan("tiny"))
+app.use(cors())
+app.use('/auth', route)
+// initalize Parse 
+Parse.initialize(PARSE_APP_ID, PARSE_JAVASCRIPT_KEY)
+Parse.serverURL = "https://parseapi.back4app.com"
+
+/*
```

Comment:
nit: remove comments with code block OR if they are needed in future commits, add a disclaimer to the top of the comment block with their future intention

Reply:
Totally forgot I did that! Thank you for pointing it out!